package.json

```diff
@@ -31,6 +31,10 @@
     "url": "https://github.com/sn-satyendra/webpack-font-preload-plugin/issues"
   },
   "homepage": "https://github.com/sn-satyendra/webpack-font-preload-plugin#readme",
+  "dependencies": {
+    "jsdom": "^16.4.0",
+    "webpack-sources": "^2.2.0"
```

Comment:
@sunnylqm : Do we need to add `webpack-sources` as a dependency as it is already a dependency of `webpack` and `webpack` is a peer dependency of this plugin?
Also `jsdom` is recently added as a peer dependency.